src/app/src/components/Filters/StyledSelect.jsx

```diff
@@ -68,42 +69,78 @@ function StyledSelect({
+                            isMulti
+                            id={name}
+                            name={name}
+                            className={`basic-multi-select ${classes.selectStyle}`}
+                            classNamePrefix="select"
+                            styles={selectFilterStyles}
+                            placeholder="Select"
+                            {...rest}
+                        />
+                    );
+                if (isSideBarSearch)
+                    return (
+                        <ReactSelect
+                            isMulti
+                            id={name}
+                            components={{
+                                DropdownIndicator: () => (
+                                    <div
+                                        style={{
+                                            display: 'flex',
+                                            marginRight: '0.5em',
+                                        }}
+                                    >
+                                        <ArrowDropDownIcon />
+                                    </div>
+                                ),
+                                IndicatorSeparator: null,
+                            }}
+                            name={name}
+                            className={`basic-multi-select notranslate ${classes.selectStyle}`}
+                            classNamePrefix="select"
+                            styles={selectFilterStyles}
+                            placeholder="Select"
+                            {...rest}
+                        />
+                    );
+                return (
+                    <ReactSelect
+                        isMulti
+                        id={name}
+                        components={{
+                            DropdownIndicator: () => (
+                                <div
+                                    style={{
+                                        display: 'flex',
+                                        marginRight: '0.5em',
+                                    }}
+                                >
+                                    <ArrowDropDownIcon />
+                                </div>
+                            ),
+                            IndicatorSeparator: null,
+                        }}
+                        name={name}
+                        className={`basic-multi-select notranslate ${classes.selectStyle}`}
+                        classNamePrefix="select"
+                        styles={selectFilterStyles}
+                        placeholder="Select"
+                        menuPosition="fixed"
+                        menuPortalTarget={document.body}
+                        closeMenuOnScroll={e =>
+                            e.target.classList === undefined ||
+                            !e.target.classList.contains('css-4ljt47-MenuList')
```

Comment:
`'css-4ljt47-MenuList'`

I think it is a dynamically generated class name, isn't it?

Reply:
No. This class is always generated by react select for the drop-down items list.